### ISSUE TITLE
Card description should be fixed to 160 chars, and fixed height

### ISSUE THREAD

**Issue by zoharbabin** [kaltura/kaltura-api-recipes#16]
Make the height fixed, so that this doesn't happen.
We should force the description field of recipes to a max chars so this doesn't happen.

The card description should not be the same as the body text in the first screen of the recipes.. it should be at length friendly for Google: 160 chars

![cookbook-length-height](https://cloud.githubusercontent.com/assets/150514/13288133/3e76d4f0-db13-11e5-945b-8f9c5c1c58b6.png)

**Comment by rbren** [comment]
I've fixed the height of the cards for now, so that the extra text just gets hidden.  Working on turning it into a scroll now.

I've changed the default text from the instructions for the first step to a list of the service/action pairs used in the recipe. We can also set the text manually using the "tip" field - see Authentication for an example:
https://github.com/kaltura/kaltura-api-recipes/blob/development/recipes/authentication.json#L6
